tools/report-converter/tests/unit/sparse_output_test_files/sample.out

```diff
@@ -0,0 +1,6 @@
+files/sample.c:4:1: warning: symbol 'machine_id' was not declared. Should it be static?
```

Comment:
I tried to run the sparse on the test files but I am not able to reproduce this output file:
```sh
$ sparse -Wsparse-all sample.c
```
```
sample.c:6:11: warning: non-ANSI function declaration of function 'main'
sample.h:7:6: warning: symbol 'refcount_inc' was not declared. Should it be static?
sample.c:4:14: warning: symbol 'machine_id' was not declared. Should it be static?
```

Can you please add a Makefile which provides the full command how we can reproduce this output with sparse easily? Similar to this one: https://github.com/Ericsson/codechecker/blob/master/tools/report-converter/tests/unit/pyflakes_output_test_files/Makefile

Reply:
@csordasmarton Yes, it does not produce the output, basically, the lines starting with `./filename` are not produced. I tried some permutations but those lines were not a part of the output. So I hardcoded a sample output file so that all the regex and features are tested properly. I also got the same output with the command above. So what should we do in this case?

Reply:
Okay now I understand it :blush: However a Makefile still would be good at this point (if someone want's to reproduce this is will be easier to understand it) and we can leave a comment there that the output in the repository contains extra outputs to test multiline reports:
```makefile
all:
    # You can write a short description here why the output is different.
    sparse -Wsparse-all sample.c > ./sample.out || exit 0;
```

Reply:
Alright I will code the Makefile and push the changes to the branch 😄